package.json

```diff
@@ -68,7 +70,8 @@
     "test:coveralls": "jest /tests/ --coverage && coveralls < coverage/lcov.info",
     "prepublishOnly": "npm run build:prod && npm run build:dev && npm run test",
```

Comment:
Perfect. Updated in the new PR I'll be submitting.

Would you like the `prepublishOnly` script to be the last one? If so, I can reorder the name of the script so that it's `build:declaration`, and is properly alphabetized with the existing `build:` scripts.

Reply:
Hmm that's a good thought. Might be worth renaming it to `build:declaration` or `build:definitions`, for the sake of consistency.
At the moment, they're not alphabetised, so I'm not overly concerned by that, but it might be good to put it either before or after the `build:grammars` script.